Pull request: FRED branch in Master branch

AndreSonntag opened the pull request:
FRED branch include the microservice for the extractor FRED
The most important class is the FredMsController.java
--

RicardoUsbeck commented:
1) Why is the "discovery client" folder empty?
2) Build failure in web client
![image](https://user-images.githubusercontent.com/4354740/33879642-5f2bb478-df2f-11e7-9211-b108fb06132f.png)

AndreSonntag commented:
During the development, I deleted the discoveryclient and webclient. I forgot that they are on the master branch. Later I added this projects to my branch again. I didn't look in the projects after the adding.

After the fixing, I'm gonna start a new pull request.
